ext/pgsql/pgsql.stub.php

```diff
@@ -951,6 +970,9 @@ function pg_exit_pipeline_mode(PgSql\Connection $connection): bool {}
     function pg_pipeline_sync(PgSql\Connection $connection): bool {}
     function pg_pipeline_status(PgSql\Connection $connection): int {}
 #endif
+
+    /** @param PgSql\Connection|int $connection */
```

Comment:
This comment is not needed any more now.
Feel free to remove this when merging if CI is happy.